test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -258,30 +262,112 @@ private static void getFailureReason() {
 
     /**
      * Position the instruction frame with testWindow (testcase created
-     * window) by the specified position.
+     * window) by the specified position. If testWindow is null, only
+     * the instruction frame is positioned according to the Position
+     * parameter.
      * Note: This method should be invoked from the method that creates
-     * testWindow.
+     * testWindow. At test-level, the testWindow must be made visible
```

Comment:
@prrace If the testWindow is made visible before calling `positionTestWindow()`, it is seen in the previous location for a short duration before being moved to the final location. Visually, the swap from previous to new location was visible. To avoid this, `testWindow.setVisible()` needs to called after `positionTestWindow()`. https://github.com/openjdk/jdk/pull/9525#issuecomment-1205915695